frontend/app/components/profile/profile.tsx

```diff
@@ -32,21 +32,32 @@ async function signout() {
 export function Profile() {
   const intl = useIntl();
   const rootRef = useRef<HTMLDivElement>(null);
+  const commentsLimit = useMemo(() => 10, []);
```

Comment:
looks unnecessary to me. it can be a simple variable or you can move it outside of the component

Reply:
yeah, it looks strange for me too now :D fixed